File: year-2/programming-design/c#/motorcycle-contest/MotorcycleContest/Domain/Entities/Race.cs
namespace MotorcycleContest.Domain.Entities
{
    public class Race : Entity<int>
    {
        public int Capacity { get; set; }

        public Race()
        {
            this.Capacity = -1;
        }

        public Race(int capacity)
        {
            this.Capacity = capacity;
        }

        public override string ToString()
        {
            return "Race " + ID + ", capacity " + Capacity + "; ";
        }
    }
}

File: year2/mpp/java/MotorcycleRaceRest/MotorcycleRestServer/src/main/java/rest/contest/domain/Race.java
package rest.contest.domain;

public class Race extends Entity<Integer> {
    private Integer capacity;
    private Integer id;

    public Race() {
        capacity = null;
    }

    public Race(Integer capacity) {
        this.capacity = capacity;
    }

    public Race(Integer capacity, Integer id) {
        this.capacity = capacity;
        this.id = id;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getCapacity() {
        return capacity;
    }

    public void setCapacity(Integer capacity) {
        this.capacity = capacity;
    }

    @Override
    public String toString() {
        return "Race " + getId() + ", capacity " + capacity + "; ";
    }
}
